api/DHSC.FingertipsNext.Modules.HealthData.UnitTests/AutoMapper/AutoMapperTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using AutoMapper;
+using DHSC.FingertipsNext.Modules.HealthData.Mappings;
+using DHSC.FingertipsNext.Modules.HealthData.Schemas;
+using DHSC.FingertipsNext.Modules.HealthData.Tests.Helpers;
+using Shouldly;
+
+namespace DHSC.FingertipsNext.Modules.HealthData.Tests.AutoMapper;
+
+public class AutoMapperTests
+{
+    readonly Mapper _mapper;
+    public AutoMapperTests() {
+        var profiles = new AutoMapperProfiles();
+        var configuration = new MapperConfiguration(cfg => cfg.AddProfile(profiles));
+        _mapper = new Mapper(configuration);
+    }
+
+    [Fact]
+    public void
+        Mapper_ShouldMapAHeathMeasure_ToAHealthDataPoint()
```

Comment:
Typo in the first instance of Hea**l**th
```suggestion
        Mapper_ShouldMapAHealthMeasure_ToAHealthDataPoint()
```